### DISCUSSION TITLE
Welcome to testify Discussions!

### DISCUSSION THREAD

**Discussion by brackendawson** [stretchr/testify#1539] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brackendawson** [comment]
Hi, I'm Bracken, a relatively new maintainer of testify. I want to merge your code, but I also want to not break things. Testify is used by nearly half a million public Go modules.

Help is always appreciated, here are some things you can do before earning access to this repo:
* Answer questions in discussions, or on our [Slack](https://gophers.slack.com/archives/C18A3680H).
* Triage issues, find duplicates etc..
* Review PRs, comments are helpful, and even if you don't get a shiny green check, your LGTM is still worth something.

**Reply by hendrywiranto** [reply]
Good to hear that testify finally has more maintainers. Goodluck! 👍
